.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+ignore =
+    E501,
+    E203,
+    W503
```

Comment:
I've only seen `E203` ignored before for `black`: https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8

Any particular reason the other two are ignored?

Reply:
W503 (line break before operators), goes against PEP8.

E501 (line too long) is just really annoying. Black takes care of line length for functional expressions, but it won't automatically break strings down. Having a string being long is not something that should get an error IMO.